starters/apps/basic/src/routes/todolist/index.tsx

```diff
@@ -1,17 +1,24 @@
 import { component$ } from '@builder.io/qwik';
-import { type DocumentHead, loader$, action$, zod$, z, Form } from '@builder.io/qwik-city';
+import {
+  type DocumentHead,
+  routeLoader$,
+  globalAction$,
+  zod$,
+  z,
+  Form,
+} from '@builder.io/qwik-city';
 
 interface ListItem {
   text: string;
 }
 
 export const list: ListItem[] = [];
 
-export const useListLoader = loader$(() => {
+export const useListLoader = routeLoader$(() => {
   return list;
 });
 
-export const useAddToListAction = action$(
+export const useAddToListAction = globalAction$(
```

Comment:
imo the replacement for this should be:
```suggestion
export const useAddToListAction = routeAction$(
```

Reply:
> imo the replacement for this should be:

fixed